pages/_app/index.jsx

```diff
@@ -3,11 +3,12 @@ import NextApp from 'next/app'
 import { withRouter } from 'next/router'
 import { autorun } from 'mobx'
 
-import { LOGIN_URL } from '../../config'
-import Route from './route'
-
 import '@oacore/design/lib/index.css'
 
+import Route from './route'
+import { getLoginPage } from '../../config'
```

Comment:
Why this is not `from 'config'`?

Reply:
because of my stupid IDE. I need to tweak settings somehow.